src/actions/UploadForm/uploadFormAction.js

```diff
@@ -0,0 +1,37 @@
+import actionNames from '../actionNames'
+
+export const updateUploadResponse = (response) => {
+    return {
+        type: actionNames.UPDATE_UPLOAD_RESPONSE,
+        payload: response
+    }
+}
+
+export const uploadFile = (data) => {
+    let url = "http://localhost:3030/upload";
+    let body = new FormData();
+    //console.log(data);

```

Comment:
Oopsies, let's get rid of this guy.

Reply:
Just a note. url and body should be assigned as const. I will also suggest that you declare global level constants in caps like so: const API_URL = "http://localhost:3000";

Reply:
I agree that url should be a constant, or added to a configuration (which we do in nephroseq).  I would argue that body shouldn't be a constant though...cuz it does change.